server/channels/store/sqlstore/migrate.go

```diff
@@ -60,7 +61,7 @@ func NewMigrator(settings model.SqlSettings, logger mlog.LoggerIFace, dryRun boo
 		return nil, fmt.Errorf("error while checking DB collation: %w", err)
 	}
 
-	engine, err := ss.initMorph(dryRun)
+	engine, err := ss.initMorph(dryRun, true)
```

Comment:
We /don't/ disable the logging for this migration test.

Reply:
This is on me: I fell into the trap you identified in the very next comment. Will fix!